checkov/terraform/checks/resource/kubernetes/ReadonlyRootFilesystem.py

```diff
@@ -0,0 +1,31 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class ReadonlyRootFilesystem(BaseResourceCheck):
```

Comment:
I'm not sure, but is it correct that this check is also added with the PR? 😄

Reply:
no that was not my intention